tasks/us/census/lodes.py

```diff
@@ -635,7 +630,7 @@ def columns(self):
     def populate(self):
         for infile in self.input()['data']:
             # gunzip each CSV into the table
-            cmd = r"gunzip -c '{input}' | psql -c '\copy {tablename} FROM STDIN " \
+            cmd = r"gunzip -c '{input}' | cut -d',' -f-52 | psql -c '\copy {tablename} FROM STDIN " \
```

Comment:
Again, add this to the helper #301 to avoid call system utils when we're able to do it with python. For example:
- gunzip could be done with gzip library
- `cut` could be done with split
- copy command could be achieved with psycopg2

Reply:
Updated https://github.com/CartoDB/bigmetadata/issues/301